behave/features/environment.py

```diff
@@ -9,14 +9,22 @@ def before_all(context):
     chrome_options = Options()
     chrome_options.binary = "bin/headless-chromium"
```

Comment:
this is the FOSS version of chrome, right? Reckon that matters, vs full-fat chrome?

Reply:
Hmm, I'll dig while waiting for errors to occur.